workflow/tests/base_results/results_workflow_simulations2.csv

```diff
@@ -115,7 +115,7 @@ house023.xml,138.233,138.233,63.089,63.089,0.0,75.144,0.0,0.0,0.0,0.0,0.0,1.892,
-house027.xml,72.565,72.565,31.685,31.685,40.88,0.0,0.0,0.0,0.0,0.0,0.0,0.436,0.0,0.0,7.85,0.993,0.0,0.0,0.0,5.947,0.218,0.485,0.927,0.0,0.0,0.0,1.724,0.0,0.0,0.447,0.338,2.514,0.105,0.0,2.114,7.587,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,19.933,0.0,17.879,0.0,3.069,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,18.776,0.0,23.222,8.564,5.23,0.0,0.0,0.0,0.0,1579.5,3648.6,3648.6,24.054,23.236,0.719,1.784,7.914,0.454,0.0,0.593,4.982,-3.982,0.0,0.0,0.383,3.36,-0.138,1.752,0.0,10.446,0.0,2.033,-8.791,-2.847,0.489,1.125,0.656,0.056,0.0,-0.113,0.132,5.429,0.0,0.0,0.095,3.842,-0.139,-0.365,-0.97,-3.474,0.0,2.575,10.726,3.101,1610.9,1574.7,10579.5,3728.4,0.0,75000.0,36000.0,0.0,24.62,91.58,33087.0,7577.0,4494.0,0.0,375.0,6506.0,550.0,250.0,4088.0,1516.0,7731.0,19084.0,3678.0,4014.0,0.0,228.0,2868.0,270.0,163.0,0.0,2277.0,2267.0,3320.0,5496.0,1758.0,2939.0,800.0
+house027.xml,73.458,73.458,31.285,31.285,42.173,0.0,0.0,0.0,0.0,0.0,0.0,0.463,0.0,0.0,7.475,0.941,0.0,0.0,0.0,5.947,0.218,0.485,0.927,0.0,0.0,0.0,1.724,0.0,0.0,0.447,0.338,2.514,0.105,0.0,2.114,7.587,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,21.172,0.0,17.932,0.0,3.069,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,19.943,0.0,21.86,8.564,5.271,0.0,0.0,0.0,0.0,1581.0,3606.3,3606.3,24.164,22.774,0.721,1.786,7.94,0.454,0.0,0.594,4.989,-4.006,0.0,0.0,0.321,3.265,-0.167,1.463,0.0,10.465,0.0,2.143,-7.25,-2.861,0.501,1.161,0.793,0.063,0.0,-0.105,0.214,5.405,0.0,0.0,0.149,3.861,-0.167,-0.276,-0.891,-3.327,0.0,2.467,8.837,3.086,1610.9,1574.7,10579.7,3728.4,0.0,75000.0,36000.0,0.0,24.62,91.58,33087.0,7577.0,4494.0,0.0,375.0,6506.0,550.0,250.0,4088.0,1516.0,7731.0,19084.0,3678.0,4014.0,0.0,228.0,2868.0,270.0,163.0,0.0,2277.0,2267.0,3320.0,5496.0,1758.0,2939.0,800.0
```

Comment:
Confirmed that house027.xml is in climate zone 3A (so the water heater now ends up in the garage).